 <launch>

 <node pkg="datmo_ros2" exec="datmo_ros2" name="datmo_ros2" output="screen">
  <param name="lidar_frame" value="laser" />
  <param name="world_frame" value="laser" />
  <param name="threshold_distance" value="0.2" />
  <param name="euclidean_distance" value="0.25" />
  <param name="pub_markers" value="true" />
 </node>
    
</launch>